src/openrct2/management/NewsItem.cpp

```diff
@@ -310,12 +310,34 @@ News::Item* News::AddItemToQueue(News::ItemType type, rct_string_id string_id, u
     return News::AddItemToQueue(type, buffer, assoc);
 }
 
+/**
+ * Checks if News::ItemType requires an assoc
+ * @return A boolean if assoc is required.
+ */
+
+bool News::CheckIfItemRequiresAssoc(News::ItemType type)
+{
+    switch (type)
+    {
+        case News::ItemType::Null:
+            return false;
+        case News::ItemType::Award:
```

Comment:
In general we would format this with all the false labels together. Rather than lots of return false.